lib/cdo/pardot.rb

```diff
@@ -469,6 +479,13 @@ def self.post_request(url, params)
 
     response = Net::HTTP.post_form(uri, params)
 
+    # Post to chat channel when known Pardot error occurs, then raise exception
+    # to be caught by calling functions.
+    if PARDOT_ERROR_CODES.include?(response.code)
+      ChatClient.message(CHAT_CHANNEL, "Pardot request failed with HTTP #{response.code}")
```

Comment:
A small suggestion: Include a link to this file in the Slack message, so when this happens in the future we don't have to track down where the error messages came from.  In general, I've been wishing our automated Slack integrations were more self-documenting.

Example:

![image](https://user-images.githubusercontent.com/1615761/57030686-6579fc00-6bfa-11e9-98ab-06cca57cb7de.png)


https://github.com/code-dot-org/code-dot-org/blob/5fd09ecb17be8435c9ce4569da94fcfb4935aea2/bin/cron/zendesk_slack_report#L140